### PR TITLE
{bio}[GCCcore/12.3.0] OBITools3 v3.0.1b26

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3129.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/59343ad2dce0e64f6ea5be3038fca44e for a full test report.

File: easybuild/easyconfigs/o/OBITools3/OBITools3-3.0.1b26-GCCcore-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    ('CMake', '3.26.3')
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('IPython', '8.14.0'),
+]
+
+use_pip = True
+
```
Comment (jfgrimm) [inline easybuild/easyconfigs/o/OBITools3/OBITools3-3.0.1b26-GCCcore-12.3.0.eb:27]:
```suggestion
use_pip = True
download_dep_fail = True

```

**Comment by jfgrimm** [comment]
@boegelbot: please test @ generoso

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node030.viking2.yor.alces.network - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/0313ac98f48e2a6590d5042be49a1d4e for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3

**Review by jfgrimm — APPROVED** [review]
lgtm

**Comment by jfgrimm** [comment]
Going in, thanks @boegel!

**Pull request merged**